src/Services/Catalog/Catalog.Presentation/Common/OptionsSetup/CategoryPaginationOptionsSetup.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Catalog.Application.Common.Options;
+using Microsoft.Extensions.Options;
+
+namespace Catalog.Presentation.Common.OptionsSetup;
+
+public class CategoryPaginationOptionsSetup : IConfigureOptions<CategoryPaginationOptions>
```

Comment:
this class is unnecessary, you can configure DI to inject IOptions<CategoryPaginationOptions>